types/react-dom/v15/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "dtslint/dt.json"
+}
```

Comment:
Always remember to add new line at the end of the file. You or someone else will thank you later, in case some tool expects a new line there and finds it 😄